utils/hosts.py

```diff
@@ -62,7 +53,7 @@ def host_name(self, env, is_api=False, port="0000", protocol="https"):
         # allow us to point to another domain if we need to
         base_domain = os.getenv("BASE_DOMAIN", "loadtest.dp3.us")
         # NOTE: deployed protocol is always https
-        return f"https://{'api' if is_api else self.deployed_value}.{base_domain}"
+        return f"https://api.{base_domain}"
```

Comment:
In the previous iteration, `is_api` was only true for the prime tests. So when running the office tests we wouldn't expect the `api` prefix as part of the host name. Is this still the case/ is there a related change that modifies the way this function is used.

Reply:
😞 I swear I had seen it on the other ones as well, but now that you called it out and I'm looking on master, you're right. I'll bring this back in. That's what I get for primarily only testing the prime api with my changes. Thanks for catching this!

Reply:
I kind of reverted it. I didn't do a direct revert because `is_api` is somewhat confusing since they're all APIs. 

Here is the commit that brings this back: https://github.com/transcom/milmove_load_testing/pull/97/commits/3bd1f6624afa560c4b619eff358f4f62549bfb73

Though I am also thinking of maybe changing how URLs are formed to be this way: https://github.com/transcom/milmove_load_testing/pull/97/commits/8196f38fa91ce6f8a43af316b3cba2400a12cb22